lib/vsopdf/vso_appointment_form.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
```

Comment:
haven't filled a pdf form myself before, but I believe this class and the related pdf should live in `lib/pdf_fill/forms` @lihanli can confirm or correct me